content/en/docs/ambient/install/multicluster/verify/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Verify the ambient installation
+description: Verify that Istio ambient mode has been installed properly on multiple clusters.
```

Comment:
```suggestion
description: Verify that Istio ambient mesh has been installed properly on multiple clusters.
```

(or "Istio in ambient mode")

this terminology remains silly and I apologise for my part in prolonging it